config/config_test.go

```diff
@@ -41,7 +40,7 @@ func Test_Parse_Checks(t *testing.T) {
 			want: config.Config{
 				Annotations: []config.Annotation{
 					{
-						Checks:  []string{"binary-artifacts"},
+						Checks:  []string{"Binary-Artifacts"},
```

Comment:
in my opinion this really should be case insensitive, so the old "binary-artifacts" should work.